w_common_tools/pubspec.yaml

```diff
@@ -0,0 +1,33 @@
+name: w_common_tools
+version: 1.21.6
```

Comment:
Reminder: we'll need to update the rosie config to auto-bump this version, as well